client/package-lock.json

```diff
@@ -17,6 +17,7 @@
         "@types/react-dom": "^16.9.8",
         "cogo-toast": "^4.2.3",
         "hookrouter": "1.2.3",
+        "lrc-kit": "^1.1.1",
```

Comment:
Try running `npm i` in the client folder and then committing `client/package-lock.json` - hopefully it should be rid of this reference.